.github/workflows/ci.yml

```diff
@@ -88,13 +88,19 @@ jobs:
           fi
           sudo apt update && sudo apt install -y "${PKGS[@]}"
 
+      - name: Install gcc-11
+        if:  matrix.compiler == 'gcc'
+        run: |
+          sudo sed -i 's/bionic/jammy/g' /etc/apt/sources.list
```

Comment:
This is wrong on so many levels (see [1]), you either upgrade the whole CI image (`runs-on: ubuntu-18.04`) or install the packages you need from somewhere else like a PPA.

[1] https://wiki.debian.org/DontBreakDebian#Don.27t_make_a_FrankenDebian

Reply:
you right, but then we need to bump it to ubuntu-20.04 if @brammool agree that we can do it.
and a ppa is needed anyway because gcc-11 is not in the ubuntu 20.04 repos